content/hi/docs/contribute/new-content/open-a-pr.md

```diff
@@ -0,0 +1,34 @@
+---
+title: एक पुल अनुरोध खोलना
+content type: संकल्पना
+weight: 10
+card:
+    name: योगदान देना
+    weight: 40
+---
+
+<!-- अवलोकन  -->
+
+{{< note >}}
+**कोड डेवलपर्स:**:यदि आप किसी के लिए एक नई सुविधा का दस्तावेजीकरण कर रहे हैं
+आगामी कुबेरनेट्स रिलीज, देखना
+[दस्तावेज़ एक नई सुविधा](/docs/contribute/new-content/new-features/)|
+{{< /note >}}
+
+नए सामग्री पृष्ठों में योगदान करने या मौजूदा सामग्री पृष्ठों में सुधार करने के लिए, पुल अनुरोध (पीआर) खोलें। सुनिश्चित करें कि आप [शुरू करने से पहले](/docs/contribute/new-content/overview/#before-you-begin) में सभी आवश्यकताओं का पालन करते हैं|
```

Comment:
```suggestion
नए सामग्री पृष्ठों में योगदान करने या मौजूदा सामग्री पृष्ठों में सुधार करने के लिए, पुल अनुरोध (PR) खोलें। सुनिश्चित करें कि आप [शुरू करने से पहले](/docs/contribute/new-content/overview/#before-you-begin) अनुभाग में सभी आवश्यकताओं का पालन करते हैं।
```

Reply:
Done! All the suggestions that you suggested were up to the mark! Thank you for reviewing and suggesting it.